monGARS/core/embeddings.py

```
@@ -160,15 +203,17 @@ async def _encode_with_huggingface(
+                except _KNOWN_MANAGER_EXCEPTIONS as exc:
                     logger.exception(
                         "llm2vec.encode.failed",
                         extra={
                             "chunk_size": len(chunk),
                             "backend_size": len(backend_payloads),
                         },
                     )
-                    raise EmbeddingBackendError("LLM2Vec encode failed") from exc
+                    raise EmbeddingBackendError(
+                        "embedding backend unavailable"
+                    ) from exc
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)